Mac Travis CI builds fail to find Qt

## Environment

* SuperCollider version: Travis CI Mac builds on Default
* Operating system: macOS
* Other details: 

## Steps to reproduce

Trigger a Travis CI build in this project, and see that the Mac build fails.

An example failure is:

https://travis-ci.org/supercollider/supercollider/jobs/605778915#L2865

```
2865  CMake Error at QtCollider/CMakeLists.txt:30 (find_package):
2866  By not providing "FindQt5.cmake" in CMAKE_MODULE_PATH this project has
2867  asked CMake to find a package configuration file provided by "Qt5", but
2868  CMake did not find one.
2869
2870  Could not find a package configuration file provided by "Qt5" (requested
2871  version 5.7) with any of the following names:
2872
2873    Qt5Config.cmake
2874    qt5-config.cmake
```

There are lots of warnings further up.

## Expected vs. actual behavior

Expected: Travis Mac CI builds pass if the code is good
Actual: Travis Mac CI builds fail to find Qt

## Other comments

There was a successful build 25 days ago, so it looks like it has failed sometime since then:
https://travis-ci.org/supercollider/supercollider/jobs/594348082?utm_medium=notification&utm_source=github_status

I first observed the problem on a build on my fork 2 days ago:
https://travis-ci.com/claremacrae/supercollider/jobs/250688604

So it may be a change in Travis Mac environment in between.

--- a/.travis.yml
+++ b/.travis.yml
@@ -18,7 +18,7 @@ matrix:
           - DO_LINT=true # only lint on fastest build
       group: edge
     - os: osx
-      osx_image: xcode8.3
+      osx_image: xcode9.4
       env:
           - QT=true
           - DO_LINT=false

--- a/.travis/before-install-osx.sh
+++ b/.travis/before-install-osx.sh
@@ -2,14 +2,14 @@
 
 export HOMEBREW_NO_ANALYTICS=1
 
-# according to https://docs.travis-ci.com/user/caching#ccache-cache
-brew install ccache
-export PATH="/usr/local/opt/ccache/libexec:$PATH"
-
 brew install libsndfile || true
 brew install portaudio || true
+brew install ccache || true
 brew install qt5 || true
 brew link qt5 --force
+
+# according to https://docs.travis-ci.com/user/caching#ccache-cache
+export PATH="/usr/local/opt/ccache/libexec:$PATH"
 
 # To get less noise in xcode output
 gem install xcpretty

--- a/.travis/qpm-prep.sh
+++ b/.travis/qpm-prep.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 cd $TRAVIS_BUILD_DIR/BUILD
-sudo pip2 install git+https://github.com/scztt/qpm.git@qpm-unit
+sudo pip2 install git+https://github.com/brianlheim/qpm.git@pip-setup
 
 mkdir $HOME/Quarks && cd $HOME/Quarks
 git clone --depth=1 https://github.com/supercollider-quarks/API

